k8s/istio-install/istio-manager.yaml

```diff
@@ -2,26 +2,26 @@
 apiVersion: v1
```

Comment:
and istio-install as controlplane ? because Istio is present even in all the apps as proxy.